src/components/shared/Form/Form.jsx

```diff
@@ -41,18 +43,33 @@ class FormComponent extends Component {
     );
   };
 
+  onFacebookButtonClick = () => {
+    window.location =
+      'https://hermes-ah-backend-staging.herokuapp.com/api/v1/auth/facebook';
+  };
+
+  onTwitterButtonClick = () => {
+    window.location =
+      'https://hermes-ah-backend-staging.herokuapp.com/api/v1/auth/twitter';
+  };
+
+  onGoogleButtonClick = () => {
+    window.location =
+      'https://hermes-ah-backend-staging.herokuapp.com/api/v1/auth/google';
+  };
+
   renderSocialLogin = () => {
     return (
       <div className="social-login">
-        <a href="htttwitter.com">
+        <Link onClick={this.onTwitterButtonClick}>
```

Comment:
I believe the onClick can stand on its own without the Link tag since it is not being used. Kindly remove the Link tag. As soon as that is done, this would be merged. Nice job